packages/util/src/environment.ts

```diff
@@ -83,6 +83,22 @@ export function isReactNative(): boolean {
   );
 }
 
+/** Detects Electron apps. */
+export function isElectron(): boolean {
+  return getUA().indexOf('Electron/') >= 0;
```

Comment:
Looks good, just thought it was an interesting coincidence [this was posted today](https://groups.google.com/a/chromium.org/forum/m/#!msg/blink-dev/-2JIRNMWJ7s/yHe4tQNLCgAJ), definitely something we need to keep an eye on!